src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -209,7 +210,7 @@ def create(style_dataset, content_dataset, style_feature=None,
     _style_loss_mult = params['style_loss_mult']
 
     num_gpus = _mxnet_utils.get_num_gpus_in_use(max_devices=params['batch_size'])
-    use_mps = _use_mps() and num_gpus == 0
+    use_mps = _use_mps() and num_gpus == 0 and (float(_platform.mac_ver()[0]) >= 10.15)
```

Comment:
We should try to keep things consistent in our code base to make it easier to understand and to update. Look for how other places in our code have solved the same problem you're trying to solve!

https://github.com/apple/turicreate/blob/037e112032c40c330885bf9cb7ba136d7a61e020/src/python/turicreate/toolkits/_internal_utils.py#L706

Reply:
Thanks for pointing out that function!